src/model/ThermalZone.cpp

```diff
@@ -2631,6 +2633,81 @@ namespace model {
+      // Check that the model has a sql file
+      if (!model().sqlFile()) {
+        LOG(Warn,
+            "This model has no sql file, cannot retrieve the ZoneSizes autosized value '" << columnName << "' with loadType '" << loadType << "'.");
+        return boost::none;
+      }
+
+      if ((loadType != "Cooling") && (loadType != "Heating")) {
+        LOG(Warn, "Cannot retrieve the ZoneSizes autosized value '" << columnName << ", Invalid loadType, must be one of ['Cooling', 'Heating']");
+        return boost::none;
+      }
+
+      // Get the object name and transform to the way it is recorded
+      // in the sql file
+      std::string sqlName = name().get();
+      boost::to_upper(sqlName);
+
+      const std::string directQuery = fmt::format(
+        R"sql(
+SELECT {} FROM ZoneSizes
+  WHERE ZoneName = ?
+    AND LoadType = ?;
+      )sql",
+        columnName);
+      boost::optional<double> val = model().sqlFile().get().execAndReturnFirstDouble(directQuery,
+                                                                                     // bindArgs
+                                                                                     sqlName, loadType);
+      if (!val) {
+        LOG(Debug, fmt::format(R"sql(The direct query failed:
+SELECT {} FROM ZoneSizes
+  WHERE ZoneName = '{}'
+    AND LoadType = '{}';)sql",
+                               columnName, sqlName, loadType));
+      }
+
+      return val;
+    }
+
+    // SQL Queries
+    boost::optional<double> ThermalZone_Impl::autosizedMaximumOutdoorAirFlowRate() const {
+      //TODO: I don't think this is available anywhere actually
+      return boost::none;
+    }
+
+    boost::optional<double> ThermalZone_Impl::autosizedMinimumOutdoorAirFlowRate() const {
+      // The same value is passed for both cooling and heating, so no need to check both
+      return getAutosizedValueFromZoneSizes("CalcOutsideAirFlow", "Cooling");
+    }
+
+    boost::optional<double> ThermalZone_Impl::autosizedCoolingDesignAirFlowRate() const {
+      return getAutosizedValueFromZoneSizes("UserDesFlow", "Cooling");
+    }
+
+    boost::optional<double> ThermalZone_Impl::autosizedHeatingDesignAirFlowRate() const {
+      return getAutosizedValueFromZoneSizes("UserDesFlow", "Heating");
+    }
+
+    boost::optional<double> ThermalZone_Impl::coolingDesignLoad() const {
+      return getAutosizedValueFromZoneSizes("UserDesLoad", "Cooling");
+    }
+
+    boost::optional<double> ThermalZone_Impl::heatingDesignLoad() const {
+      return getAutosizedValueFromZoneSizes("UserDesLoad", "Heating");
+    }
+
+    boost::optional<double> ThermalZone_Impl::designAirFlowRate() const {
+      auto coolingFlow_ = autosizedCoolingDesignAirFlowRate();
+      auto heatingFlow_ = autosizedHeatingDesignAirFlowRate();
+      if (coolingFlow_ && heatingFlow_) {
+        return coolingFlow_.get() + heatingFlow_.get();
```

Comment:
I think this should be a maximum of the two, not a sum.  This is what we have in openstudio-standards:
https://github.com/NREL/openstudio-standards/blob/master/lib/openstudio-standards/hvac_sizing/Siz.ThermalZone.rb#L223-L224

```
# Use maximum of the two as actual design flow rate
result = OpenStudio::OptionalDouble.new([clg_des_air_flow_rate.to_f, htg_des_air_flow_rate.to_f].max)
```

Reply:
Agree, definitely not the sum.

Reply:
Done in 66dbd48c98bbc67897051301a7d85a988faa6484 , thanks for the catch!